asst1/asst1.py

```diff
@@ -0,0 +1,24 @@
+from collections import Counter
+
```

Comment:
honestly great code! very pythonic (: would you mind if we presented it at the comp meeting?